inst/include/dust/dust.hpp

```diff
@@ -73,8 +73,10 @@ class Dust {
     }
   }
 
-// This could be removed/commented out eventually
-#ifdef __NVCC__
+  // We only need a destructor when running with cuda profiling; don't
+  // include ond otherwise because we don't actually follow the rule
+  // of 3/5/0
+#ifdef DUST_ENABLE_CUDA_PROFILER
```

Comment:
Is the `#endif` missing here?

Reply:
on line 83, no?

Reply:
Yes, I think was just obscured by the diff!